src/prefs/TracksBehaviorsPrefs.cpp

```diff
@@ -110,18 +107,11 @@ void TracksBehaviorsPrefs::PopulateOrExchange(ShuttleGui & S)
                     {wxT("/GUI/CircularTrackNavigation"),
                      false});
       S.TieCheckBox(XXO("&Type to create a label"),
-                    {wxT("/GUI/TypeToCreateLabel"),
+                     {wxT("/GUI/TypeToCreateLabel"),
                      false});
       S.TieCheckBox(XXO("Use dialog for the &name of a new label"),
                     {wxT("/GUI/DialogForNameNewLabel"),
                      false});
-#ifdef EXPERIMENTAL_SCROLLING_LIMITS
-      S.TieCheckBox(XXO("Enable scrolling left of &zero"),
-                    ScrollingPreference);
-#endif
-      S.TieCheckBox(XXO("Advanced &vertical zooming"),
-                    {wxT("/GUI/VerticalZooming"),
-                     false});
```

Comment:
Maybe you should remove one preference at a time in each commit, and do each really completely.

"/GUI/AdjustSelectionEdges" is used in `ViewInfo::UpdatePrefs` so there is the same objection that it's possible to have behavior frozen one or another way depending on the config file.  Admittedly it's unlikely anyone would do this.  Still it leaves the code simplification incomplete.

So you should also eliminate the member variable `ViewInfo::bAdjustSelectionEdges` and simplify by assuming it was true wherever it was used before.